src/tests/ftest/osa/online_extend.py

```diff
@@ -142,7 +142,10 @@ def run_online_extend_test(self, num_pool, racer=False,
             display_string = "Pool{} space at the End".format(val)
             self.pool = pool[val]
             self.pool.display_pool_daos_space(display_string)
-            self.run_ior_thread("Read", oclass, test_seq)
+            if app_name == "ior":
+                self.run_ior_thread("Read", oclass, test_seq)
+            else:
+                break
```

Comment:
Why `break`? Should we just skip this `for` loop altogether if `app_name != "ior"`?

Reply:
pool_cont_dict[] will be set inside run_ior_thread(), i.e.  if ior is not being run, then the following line "self.container = self.pool_cont_dict[self.pool][0]" will fail.

> Should we just skip this for loop altogether if app_name != "ior"?

Yes, probably. Though I would still expect the container check will be done in this case, but that probably need extra change for setting pool_cont_dict[], which probably can be done in other patches.

Reply:
For now, I would think it makes sense to either (1) skip the entire `for` loop or (2) use `continue` instead so `display_pool_daos_space` is ran for each pool. By using `break`, the space is displayed for the first pool only